src/app/clients/add/submitFormHelpers.ts

```diff
@@ -225,12 +208,16 @@ const revertClientInsert = async (primaryKey: string): Promise<void> => {
     await supabase.from("clients").delete().eq("primary_key", primaryKey);
 };
 
-// TODO VFB-22: Handle error from Family Updates
 const revertClientUpdate = async (initialRecords: ClientDatabaseUpdateRecord): Promise<void> => {
-    await supabase
+    const { error } = await supabase
         .from("clients")
         .update(initialRecords)
         .eq("primary_key", initialRecords.primary_key);
+    if (error) {
+        throw new Error(
+            "We could not revert the incomplete client update at this time. Please try again later."
```

Comment:
Would this be user friendly? Since most of them might not know why client update needs to be reverted